docs/the_black_code_style/future_style.md

```diff
@@ -139,6 +139,28 @@ foo([
 ])
 ```
 
+This also applies to list unpacking:
+
+```python
+foo(
+    *[
+        1,
```

Comment:
This example isn't very compelling, as it could equivalently be written as `foo(1, 2, 3)`. I'd argue that the previous formatting is better, because it emphasizes that something weird is going on.

A more convincing variant might be if the argument is a comprehension, e.g. `foo(*[str(i) for i in range(3)])`. How do we handle that?

Reply:
Good point! Comprehensions work as any list would, so your example fitting on one line would not be formatted to anything. But I can come up with a suitable one for the docs.